src/main/java/no/unit/nva/model/instancetypes/ReportContent.java

```diff
@@ -0,0 +1,82 @@
+package no.unit.nva.model.instancetypes;
+
+import no.unit.nva.model.exceptions.InvalidPageTypeException;
+import no.unit.nva.model.pages.MonographPages;
+import no.unit.nva.model.pages.Pages;
+import no.unit.nva.model.pages.Range;
+import nva.commons.utils.JacocoGenerated;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Objects;
+
+import static java.util.Objects.nonNull;
+
+public abstract class ReportContent implements PublicationInstance {
+    public static final String SETTING_PEER_REVIEWED_FALSE =
+            "Setting peerReviewed to false as {} is assumed to not be peer-reviewed";
+    protected static final Logger logger = LoggerFactory.getLogger(ReportContent.class);
+
+    private Pages pages;
+    private boolean peerReviewed;
+
+    /**
+     * This constructor ensures that the peerReviewed value is always false.
+     *
+     * @param pages         the Pages of the PublicationInstance.
+     * @param peerReviewed  the value is always ignored.
+     * @throws InvalidPageTypeException if the type of Pages is incompatible with the PublicationInstance type.
+     */
+    @SuppressWarnings("PMD.ConstructorCallsOverridableMethod")
```

Comment:
Somehow it seems that PublicationInstance could be an abstract class. That could maybe simplify the code a bit. There is no class that practically extends another class that is not PublicationInstance and itself is a PublicationInstance. 

If PublicationInstance were an abstract class, then some "anti-patterns"  (although too harsh of a characterization here) could be avoided and some classes could become smaller.  
This is not to be solved here, but should we create an issue for this?

Reply:
I made a task for this :)